dask/array/reductions.py

```diff
@@ -37,11 +37,92 @@ def divide(a, b, dtype=None):
     return f(a, b, dtype=dtype)
 
 
-def reduction(x, chunk, aggregate, axis=None, keepdims=None, dtype=None,
-              split_every=None, combine=None, name=None, out=None):
+def reduction(x, chunk, aggregate, axis=None, keepdims=False, dtype=None,
+              split_every=None, combine=None, name=None, out=None,
+              concatenate=True, output_size=1):
     """ General version of reductions
 
-    >>> reduction(my_array, np.sum, np.sum, axis=0, keepdims=False)  # doctest: +SKIP
+    Parameters
+    ----------
+    x: Array
+        Data being reduced along one or more axes
+    chunk: callable(x_chunk, axis, keepdims)
+        First kernel function to be executed when resolving the dask graph.
```

Comment:
The term `kernel` might not be well understood by users.  I wonder if we can use the term `function` instead throughout this docstring.  I suspect that this will be more familiar to novice users.